requirements_docker.txt

```diff
@@ -5,7 +5,7 @@ natsort
 numpy
 opencv-python==3.4.2.17
 openpyxl
-pandas>=0.25.0
+pandas>=1.0.2
```

Comment:
This requirement file is actually for running inside a docker container on IBM power 9. I had to use the old pandas version to be able to install all the dependency without conflicts. Does the current test also test running in power 9 environment? If not, maybe we can keep this as it is for now?

Reply:
I don't see need to run pysero on IBM power 9, given its simplicity. It will be good to have the repo ready to use in more widely available platforms.

Reply:
Oops, I just realized I still had my github send me notifications to my biohub mailaddress which doesn't exist anymore! Sorry about that, that's why I haven't been responsive.

I just changed both requirements for consistency, happy to change back the docker requirements to make it work with the IBM servers.